cli/npm/resolution/mod.rs

```diff
@@ -205,6 +206,21 @@ pub struct NpmResolutionPackage {
   pub dependencies: HashMap<String, NpmPackageId>,
 }
 
+impl std::fmt::Debug for NpmResolutionPackage {
+  fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+    // custom debug implementation for deterministic output in the tests
+    f.debug_struct("NpmResolutionPackage")
+      .field("pkg_id", &self.pkg_id)
+      .field("copy_index", &self.copy_index)
+      .field("dist", &self.dist)
+      .field(
+        "dependencies",
+        &self.dependencies.iter().collect::<BTreeMap<_, _>>(),
```

Comment:
It would be nice if instead we could add an attribute to the property so it does this behaviour. Is it possible? Maybe that would require a custom `Debug` attribute?

Reply:
I don't know about such a way